crates/viewer/re_space_view_graph/src/ui/selection.rs

```diff
@@ -0,0 +1,86 @@
+use re_space_view::{view_property_component_ui, view_property_component_ui_custom};
+use re_types::{
+    blueprint::components::Enabled, Archetype, ArchetypeReflectionMarker, Component as _,
+};
+use re_viewer_context::{ComponentFallbackProvider, SpaceViewId, SpaceViewState, ViewerContext};
+use re_viewport_blueprint::ViewProperty;
+
+pub fn view_property_force_ui<A: Archetype + ArchetypeReflectionMarker>(
```

Comment:
I dont understand this function. What is it doing differently from `view_property_ui`?